server/data/projects.js

```diff
@@ -93,6 +93,23 @@ const createProject = async (projectObjParam, user) => {
 	);
 	return createdProject;
 };
+const likeProject = async (user, project) => {
+	const userInfo = user;
+	const userId = userInfo._id;
+	const projectId = isValidObjectId(project);
+	isValidObjectId(userId);
+	const projectCollection = await projects();
+	const getProjectInfo = await getProjectById(projectId);
+	if (!getProjectInfo) {
+		throw internalServerErr('Project could not be found');
+	}
+	getProjectInfo.likes.push(ObjectId(userId));
+	const likeProjectAcknowledgment = await projectCollection.updateOne(
+		{ _id: ObjectId(projectId) },
+		{ $set: { likes: getProjectInfo.likes } }
+	);
+	return likeProjectAcknowledgment;
```

Comment:
you cannot return the acknowledgment from here, you are sending this directly into the API response. Return the new likes array